### DISCUSSION TITLE
"'module' object is not callable"

### DISCUSSION THREAD

**Discussion by Normanras** [domdfcoding/whiptail#39] [Q&A] [answered]
This sounds super naive and basic, but I can't seem to get the dialog box to pop up. I've imported `whiptail` and within one of my functions I've tried calling `msgbox` and I keep getting errors. I can confirm that Whiptail is installed and if I call it from the terminal via bash, it loads as expected. 

Is there something basic with calling the class and functions of the class that I need to set up after installing, and importing into the script?

**Comment by domdfcoding** [comment, marked as answer]
I'm not entirely sure what's going wrong. I've added [an example](https://whiptail.readthedocs.io/en/latest/example.html) to the documentation. Can you try running that and see if you still get an error?

**Reply by Normanras** [reply]
Perfect. Thank you for the example it cleared it up. As usual with Python libraries, there's always some combination of `import` that I haven't tried. 

`from whiptail import Whiptail` was the answer and the rest of my script worked without issue! 

Thanks for the quick addition of an example.
